ui/raidboss/data/06-ew/trial/rubicante-ex.ts

```diff
@@ -0,0 +1,243 @@
+import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  getFourfold: boolean;
+  getTwinfold: boolean;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.MountOrdealsExtreme,
+  timelineFile: 'rubicante-ex.txt',
+  initData: () => {
+    return {
+      getFourfold: false,
+      getTwinfold: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'RubicanteEx Inferno',
+      type: 'StartsUsing',
+      netRegex: { id: '7D2C', source: 'Rubicante', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'RubicanteEx Shattering Heat',
+      type: 'StartsUsing',
+      netRegex: { id: '7D2D', source: 'Rubicante' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'RubicanteEx Spike of Flame',
+      type: 'StartsUsing',
+      netRegex: { id: '7D02', source: 'Rubicante' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'RubicanteEx Fourfold Flame',
+      type: 'StartsUsing',
+      netRegex: { id: '7D03', source: 'Rubicante', capture: false },
+      condition: (data) => !data.getFourfold,
+      infoText: (_data, _matches, output) => output.group!(),
+      run: (data) => data.getFourfold = true,
```

Comment:
For future reference, you can use `suppressSeconds: 5` or something here to keep triggers from re-firing rather than adding data for it.